submit.py

```diff
@@ -152,7 +158,7 @@ def get_url(cfg, option, default):
 def get_config():
     """Returns a ConfigParser object for the .kattisrc file(s)
     """
-    cfg = configparser.ConfigParser()
+    cfg = ConfigParser()
```

Comment:
On 2016-01-26 22:16, Mikael Goldmann wrote:
> 
> OK, can do that. Are you OK with the rest?

Yes, looks fine, assuming you've tested (in both Python 2 and 3).